src/main/java/com/lothrazar/cyclicmagic/util/UtilPlaceBlocks.java

```diff
@@ -50,54 +50,50 @@ public static boolean placeStateSafe(World world, @Nullable EntityPlayer player,
       return false;
     }
     IBlockState stateHere = null;
-    if (player != null && PermissionRegistry.hasPermissionHere(player, placePos) == false) {
+    if (player != null && !PermissionRegistry.hasPermissionHere(player, placePos)) {
       return false;
     }
-    if (world.isAirBlock(placePos) == false) {
+    if (!world.isAirBlock(placePos)) {
       // if there is a block here, we might have to stop
       stateHere = world.getBlockState(placePos);
       if (stateHere != null) {
         Block blockHere = stateHere.getBlock();
-        if (blockHere.isReplaceable(world, placePos) == false) {
+        if (!blockHere.isReplaceable(world, placePos)) {
```

Comment:
personal preference, but Honestly i prefer the " == false)" style, because I often skim through code, and if im on my laptop or a small screen its hard to see, i often miss seeing the "!" and i have to double check.  

so if im checking true i just go if(stuff) and checking false its if(stuff == false)

Reply:
will revert